test/python/pulse/test_cmd_def.py

```diff
@@ -138,7 +138,7 @@ def test_build_cmd_def(self):
         defaults = self.backend.defaults()
         cmd_def = defaults.build_cmd_def()
 
-        cx_pv = cmd_def.get('cx', (0, 1), P2=0)
+        cx_pv = cmd_def.get('cx-parametrized', (0, 1), P2=0)
```

Comment:
I think this is an invalid name now according to the new naming convention. We should stick to `[a-zA-Z][a-zA-Z0-9_]*`.

Reply:
ah. what do you recommend for this? I still haven't totally figured out how params work here so I was trying to write parameter-less CXs. I guess I should update CX to take a param and then this test should work as is

Reply:
`cx_parameterized` is fine.